src/Simplex/Messaging/Agent.hs

```diff
@@ -39,6 +39,7 @@ module Simplex.Messaging.Agent
     getSMPAgentClient,
     createConnection,
     joinConnection,
+    letConfirmation,
```

Comment:
confirmConnection would be more consistent

Reply:
will rename to `allow`

Reply:
done